include/RevRegFile.h

```diff
@@ -335,6 +338,42 @@ class RevRegFile {
     }
   }
 
+  /// Get a CSR register
+  template<typename T>
+  T GetCSR( size_t csr ) const {
+    // We store fcsr separately from the global CSR
+    switch( csr ) {
+    case 1: return static_cast<uint32_t>( fcsr ) >> 0 & 0b00011111u; break;
+    case 2: return static_cast<uint32_t>( fcsr ) >> 5 & 0b00000111u; break;
+    case 3: return static_cast<uint32_t>( fcsr ) >> 0 & 0b11111111u; break;
```

Comment:
There are 3 virtual FP CSR registers -- the exception flags, the rounding mode, and both combined.